cfme/automate/import_export.py

```diff
@@ -39,9 +40,7 @@ def is_displayed(self):
 
 
 class AutomateImportExportView(AutomateImportExportBaseView):
-    @View.nested
-    class flash(V2VFlashMessages):
-        ROOT = 'div.import-flash-message'
+    flash = ImportExportFlashMessages()
```

Comment:
This should still be a nested View:
```
flash = View.nested(ImportExportFlashMessages)
```

Also, `FileImportSelectorView` and `AutomateImportExportBaseView` still use `V2VFlashMessages`. Should they be updated to use `ImportExportFlashMessages` as well?